### PR TITLE
Modify to convert *sbom-info*.yaml

### PR DESCRIPTION
**Pull request by bjk7119**
## Description

- Modify to convert *sbom-info*.yaml

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/fosslight_util/parsing_yaml.py [hunk 1]
```diff
@@ -7,11 +7,14 @@
 import logging
 import codecs
 import os
+import re
 import sys
 from .constant import LOGGER_NAME
 from .oss_item import OssItem
 
 _logger = logging.getLogger(LOGGER_NAME)
+SUPPORT_OSS_INFO_FILES = ["oss-pkg-info.yaml", "sbom-info.yaml"]
```
Comment (soimkim) [inline src/fosslight_util/parsing_yaml.py:16]: `"oss-pkg-info[\s\S]*.yaml",  r"sbom-info[\s\S]*.yaml"`
Shouldn't it be like this?

File: src/fosslight_util/parsing_yaml.py [hunk 2]
```diff
@@ -7,11 +7,14 @@
...
+EXAMPLE_OSS_PKG_INFO_LINK = "https://github.com/fosslight/fosslight_prechecker/blob/main/tests/convert/oss-pkg-info.yaml"
```
Comment (soimkim) [inline src/fosslight_util/parsing_yaml.py:18]: It is suggested to rename the file fosslight_prechecker/blob/main/tests/convert/oss-pkg-info.yaml to sbom-info.yaml and also change this link.

**Review by soimkim — CHANGES_REQUESTED** [review]

File: src/fosslight_util/parsing_yaml.py [hunk 3]
```diff
@@ -46,27 +50,28 @@ def parsing_yml(yaml_file, base_path):
...
                     idx += 1
+    except AttributeError as ex:
+        _logger.error(f"Not supported yaml file format {ex}")
     except yaml.YAMLError:
         _logger.warning(f"Can't parse yaml - skip to parse yaml file: {yaml_file}")
     return oss_list, set(license_list)
 
 
-def find_all_oss_pkg_files(path_to_find, file_to_find):
+def find_sbom_yaml_files(path_to_find):
```
Comment (soimkim) [inline src/fosslight_util/parsing_yaml.py:60]: Make sure there are no Scanners using `find_all_oss_pkg_files`.

**Review by soimkim — APPROVED** [review]

File: src/fosslight_util/parsing_yaml.py (hunk 3 above)
Comment (bjk7119) [reply src/fosslight_util/parsing_yaml.py:60]: It is not used anywhere else. only used in FL Prechecker

**Pull request merged**